### PR TITLE
Issue #9485: updated example of AST for TokenTypes.LITERAL_NATIVE

### PR DESCRIPTION
**Pull request by AbhinavRajputEXE**
Fixes Issue #9485 example of AST for TokenTypes.LITERAL_NATIVE

### CODE REVIEW

**Comment by romani** [comment]
please squash commits in one.

**Comment by AbhinavRajputEXE** [comment]
@romani In the failing CIs error log it is saying that my commit message is wrong, but I rechecked it and it's following all the commit message rules. what am I doing wrong please help?
Screenshot (447)
s.

File: src/main/java/com/puppycrawl/tools/checkstyle/api/TokenTypes.java [hunk 1]
```diff
@@ -1875,6 +1875,46 @@
     /**
      * The {@code native} keyword.
      *
+     * <p>For example:</p>
+     * <pre>
+     * native void equal(int a, int b)
```
Comment (romani) [inline src/main/java/com/puppycrawl/tools/checkstyle/api/TokenTypes.java:1880]: `native void foo(){}`

Focus on modifier, all other is not relevant.

**Review by romani — CHANGES_REQUESTED** [review]
Items

File: src/main/java/com/puppycrawl/tools/checkstyle/api/TokenTypes.java (hunk 1 above)
Comment (AbhinavRajputEXE) [reply src/main/java/com/puppycrawl/tools/checkstyle/api/TokenTypes.java]: @romani I've made the requested changes, please review

**Review by romani — APPROVED** [review]

**Pull request merged**